luskan/src/test/java/org/opencadc/luskan/SiteLocationsConverterTest.java

```diff
@@ -0,0 +1,171 @@
+
+import ca.nrc.cadc.tap.TapQuery;
+import ca.nrc.cadc.tap.schema.TapSchema;
+import ca.nrc.cadc.util.Log4jInit;
+import ca.nrc.cadc.util.MultiValuedProperties;
+import ca.nrc.cadc.uws.Parameter;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SiteLocationsConverterTest {
+    private static final Logger log = Logger.getLogger(SiteLocationsConverterTest.class);
+
+    static {
+        Log4jInit.setLevel("org.opencadc.luskan", Level.INFO);
+    }
+
+    private static final TapSchema tapSchema = TestUtil.loadTapSchema();
+
+    private static final String selectList =
+        "inventory.artifact.uri, inventory.artifact.uribucket, inventory.artifact.contentchecksum, "
+            + "inventory.artifact.contentLastmodified, inventory.artifact.contentlength, "
+            + "inventory.artifact.contenttype, inventory.artifact.contentencoding, inventory.artifact.lastmodified, "
+            + "inventory.artifact.metachecksum, inventory.artifact.id ";
+    private static final String fromClause = "FROM inventory.artifact";
+    private static final String isNotNullConstraint = "(inventory.artifact.sitelocations IS NOT NULL)";
+
+    @Test
+    public void testSelectAll() {
+        String query = "select * from inventory.artifact";
+        String expected = "SELECT " + selectList + fromClause;
+        doTest(query, expected, null);
+    }
+
+    @Test
+    public void testSelectColumns() {
+        String query = "select uri, lastmodified from inventory.artifact";
+        String expected = "SELECT uri, lastModified " + fromClause + " WHERE " + isNotNullConstraint;
+        doTest(query, expected, "true");
+    }
+
+    @Test
+    public void testSelectColumnWithAlias() {
+        String query = "select uri as \"foo\" from inventory.artifact";
+        String expected = "SELECT uri as \"foo\" " + fromClause + " WHERE " + isNotNullConstraint;
+        doTest(query, expected, "true");
+    }
+
+    @Test
+    public void testSelectWithWhere() {
+        String query = "select uri from inventory.artifact where uribucket = 1";
+        String expected = "SELECT uri " + fromClause + " WHERE (uribucket = 1) and " + isNotNullConstraint;
+        doTest(query, expected, "true");
+    }
+
+    private void doTest(final String query, final String expected, final String isStorageSite) {
+        try {
+            TestUtil.job.getParameterList().clear();
+            List<Parameter> params = new ArrayList<Parameter>();
+            params.add(new Parameter("QUERY", query));
+            log.debug("query: " + query);
+            TapQuery tq = new TestQuery(isStorageSite);
+            tq.setTapSchema(tapSchema);
+            TestUtil.job.getParameterList().addAll(params);
+            tq.setJob(TestUtil.job);
+            log.debug("expected: " + expected);
+            String sql = tq.getSQL();
+
```

Comment:
I would add a `log.info("actual: " + sql)` and probably change the previous (log query and expected) to info as well.

Reply:
Updated the logging as above.